lib/publiccloud/ssh_interactive.pm

```diff
@@ -29,10 +29,13 @@ sub ssh_interactive_tunnel {
     my $upload_host = testapi::host_ip();
 
     $instance->run_ssh_command(
-        cmd => "'rm -rf /dev/sshserial; mkfifo -m a=rwx /dev/sshserial; tail -fn +1 /dev/sshserial' | tee /dev/$serialdev ", # Create /dev/sshserial fifo on remote and tail|tee it to /dev/$serialdev on local
-        timeout  => 0,    # This will also cause script_run instead of script_output to be used so the test will not wait for the command to end
+        # Create /dev/sshserial fifo on remote and tail|tee it to /dev/$serialdev on local
+        #   timeout => switches to script_run instead of script_output to be used so the test will not wait for the command to end
+        #   tunnel the worker port (for downloading from data/ and uploading assets / logs
+        cmd => "'rm -rf /dev/sshserial; mkfifo -m a=rwx /dev/sshserial; tail -fn +1 /dev/sshserial' 2>&1 | tee /dev/$serialdev; clear",
```

Comment:
This is super cryptic and hard to understand. I understand this is a quickfix, but can we please add a comment before like

    # TODO: Separate this into ssh command to run on the host and a assert_script_run to run on the openQA instance.

Other than that LGTM :rocket:

Reply:
No, you cannot `assert_script_run` much there. This command keeps running in the background on the `tunnel` console.

Reply:
Of course this can be improved. But it would not be that easy :smile:

Reply:
I understand.